libafl/src/schedulers/weighted.rs

```diff
@@ -127,10 +129,18 @@ where
             map_observer_handle: map_observer.handle(),
             last_hash: 0,
             table_invalidated: true,
+            cycle_schedules: false,
             phantom: PhantomData,
         }
     }
 
+    /// Cycle the `PowerSchedule` on completion of a queue cycle
+    #[must_use]
+    pub fn cycle_schedules(mut self) -> Self {
```

Comment:
Shouldn't this be something like `cycling_shedulers` or so?

Reply:
IMO this is more consistent with `track_novelties` or `track_indices`

Reply:
Those are methods, not Constructors, though, right? I feel like there's a difference (but I will also accept this name if you prefer)

Reply:
They return Self tho. That's why I feel like this is more consistent

Reply:
The methods only return `Self` because we hack the type system, it's really just setting the value and changing the type - it's not a Constructor (they eat a `self`)

Reply:
See https://github.com/AFLplusplus/LibAFL/blob/03d8d2eb08b5112d5476949a06b90b13849d738e/libafl/src/observers/map/mod.rs#L129

Reply:
Ok I can change it to `cycling_scheduler`